lib/rules/require-spaces-in-for-statement.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Requires spaces inbetween for statement.
+ *
+ * Type: `Boolean`
+ *
+ * Values: `true` to requires spaces inbetween for statement.
+ *
+ * #### Example
+ *
+ * ```js
+ * "requireSpacesInForStatement": true
+ * ```
+ *
+ * ##### Valid
+ *
+ * ```js
+ * for(var i = 0; i<l; i++) {
+ *     x++;
+ * }
+ * ```
+ *
+ * ##### Invalid
+ *
+ * ```js
+ * for(var i = 0;i<l;i++) {
+ *     x++;
+ * }
+ * ```
+ *
+ * ```js
+ * for(var i = 0; i<l;i++) {
+ *     x++;
+ * }
+ * ```
+ *
+ * ```js
+ * for(var i = 0;i<l; i++) {
+ *     x++;
+ * }
+ * ```
+ */
+
+var assert = require('assert');
+
+module.exports = function() {};
+
+module.exports.prototype = {
+    configure: function(requireSpacesInForStatement) {
+        assert(
+            requireSpacesInForStatement === true,
+            'requireSpacesInForStatement option requires true value or should be removed'
+        );
+    },
+
+    getOptionName: function() {
+        return 'requireSpacesInForStatement';
+    },
+
+    check: function(file, errors) {
+        file.iterateNodesByType('ForStatement', function(node) {
+            if (node.test) {
```

Comment:
I guess you can use assertion API here, should much cleaner
module called "<a href="https://github.com/jscs-dev/node-jscs/blob/8bd2ffd7bd1fc53a49d017f06322b178124534fa/lib/token-assert.js">token-assert</a>",
use example - https://github.com/jscs-dev/node-jscs/blob/8bd2ffd7bd1fc53a49d017f06322b178124534fa/lib/rules/validate-parameter-separator.js#L67-72